tests/test_expense_tools.py

```
@@ -2,10 +2,9 @@
 Unit tests for expense management tools in Zoho Books MCP Integration Server.
 """
 
-import json
-import unittest
+import pytest
 from datetime import date
-from unittest.mock import patch, MagicMock
+from unittest.mock import patch, AsyncMock
```

Ruff (0.12.2)

`unittest.mock.AsyncMock` imported but unused

Remove unused import: `unittest.mock.AsyncMock`

(F401)